frontend/src/lib/lemon-ui/LemonRadio/LemonRadio.tsx

```diff
@@ -0,0 +1,46 @@
+import clsx from 'clsx'
+
+import { IconCheckmark, IconRadioButtonUnchecked } from '../icons'
+
+export interface LemonRadioOption<T extends React.Key> {
+    label: string | JSX.Element
+    value: T
+}
+
+export interface LemonRadioProps<T extends React.Key> {
+    value?: T
+    onChange: (newValue: T) => void
+    options: LemonRadioOption<T>[]
+    disabled?: boolean
```

Comment:
We shouldn't add `disabled` here. Let's only support `disabledReason` in new components. Otherwise it's easy to forget about any indication _why_ an element is disabled, which can be really frustrating as a user